packages/pigeon/lib/cpp_generator.dart

```diff
@@ -7,6 +7,9 @@ import 'functional.dart';
 import 'generator_tools.dart';
 import 'pigeon_lib.dart' show Error;
 
+/// Documentation open symbol.
+const String openDoc = '///';
```

Comment:
For C++ this should just be `//`; see https://google.github.io/styleguide/cppguide.html#Comment_Style